run_tribler.py

```diff
@@ -1,30 +1,75 @@
 import os
 import sys
 import multiprocessing
+import logging.config
 
 from check_os import check_environment
 
+
+def setup_logging():
+    """
+    Setup logging to write logs to files inside \
+    .Tribler directory in each platforms
+    """
+    # First check if logger.conf is present or not
+    if getattr(sys, 'frozen', False):
+        # If the application is run as a bundle, the pyInstaller bootloader
+        # extends the sys module by a flag frozen=True and sets the app
+        # path into variable _MEIPASS'.
+        application_path = os.environ.get("_MEIPASS", os.path.abspath("."))
+    else:
+        application_path = os.path.dirname(os.path.abspath(__file__))
+
+    log_config = os.path.join(application_path, "logger.conf")
+
+    if not os.path.exists(log_config):
+        print "Log configuration file not found at location '%s'" % log_config
+        return
+
+    if sys.platform == 'linux2' or sys.platform == 'darwin':
```

Comment:
I think it would be better to use [platform.system](https://docs.python.org/2/library/platform.html#platform.system) here, for generality.

Reply:
Actually, you could even do:
```python
log_directory = os.path.abspath(os.environ.get('APPDATA', os.path.expanduser('.')))
```
and skip this entire statement.

Reply:
Yes, that is neat. I'll make the changes.

`log_directory = os.path.abspath(os.environ.get('APPDATA', os.path.expanduser('.')))`
Instead of `os.path.expanduser('.')` it should be `os.path.expanduser('~')` as we create the directory inside user home.

Reply:
@xoriole hmm I just checked, I guess neither is really needed:
```python
>>> os.path.abspath(os.path.expanduser('.'))
'/home/quinten'
>>> os.path.abspath(os.path.expanduser('~'))
'/home/quinten'
>>> os.path.abspath(os.path.expanduser(''))
'/home/quinten'
```

Reply:
@qstokkink It might probably be that you are running the python console from your home directory. This is my output on a console initialized at a different directory
```
>>> os.path.abspath(os.path.expanduser('.'))
'/home/sandip/personal/tribler'
>>> os.path.abspath(os.path.expanduser('~'))
'/home/sandip'
>>> os.path.abspath(os.path.expanduser(''))
'/home/sandip/personal/tribler'

```

Reply:
@xoriole good catch 👍